routes/api/assignments.js

```diff
@@ -0,0 +1,70 @@
+const express = require('express');
+const router = express.Router();
+const Assignment = require('../../models/Assignment');
+const { check, validationResult } = require('express-validator');
+const jwt = require('jsonwebtoken');
+const config = require('config');
+const bcrypt = require('bcryptjs');
+const auth = require('../../middleware/auth');
+
+// @route 	POST api/assignments
+// @desc	Route to create new assignments
+// @access 	Public
+router.post(
+    '/',
+    [
+        check('name', 'Name is required')
+            .not()
+            .isEmpty(),
+        check('detail', 'Detail is required')
+            .not()
+            .isEmpty(),
+        check('assignedInstructors', 'Assigned Instructors are required')
+            .not()
+            .isEmpty(),
+        check('dueDate', 'Please include a valid due date')
+            .not()
+            .isEmpty(),
+    ],
+    async (req, res) => {
+        const errors = validationResult(req);
```

Comment:
Does this check authentication right now?

Reply:
I see it returns a jwt with assignment.save(), but is that save function validating the token that a user should have to create a new assignment?

Reply:
I added authentication in the latest commit.